plugins/@grouparoo/sqlite/src/lib/export/exportProfile.ts

```diff
@@ -37,11 +37,11 @@ export const exportProfile: ExportProfilePluginMethod = async ({
         );
       }
       // delete
-      const query = `DELETE FROM ${table} WHERE ${primaryKey} = ${newProfileProperties[primaryKey]}`;
+      const query = `DELETE FROM "${table}" WHERE ${primaryKey} = ${newProfileProperties[primaryKey]}`;
```

Comment:
While we are on the topic of escaping, I'm wondering if /all/ user provided values should be quoted?  What if `newProfileProperties[primaryKey]` in this case was a UUID?  Would it fail?

> DELETE FROM users WHERE uuid = abc123